src/screens/Home.svelte

```diff
@@ -11,15 +11,17 @@
 <div class="home">
 	<Container>
 		<h1>Home</h1>
-		{#if $ulist.length == 1}
-			You are the only user online.
-		{:else if $ulist.length == 0}
-			Nobody is online.
-		{:else}
-			There are currently {$ulist.length} users online{#if $ulist.length}{" "}({$ulist.join(
-					", "
-				)}){/if}.
-		{/if}
+		<div class="overflow">
+			{#if $ulist.length == 1}
+				You are the only user online.
+			{:else if $ulist.length == 0}
+				Nobody is online.
+			{:else}
+				There are currently {$ulist.length} users online{#if $ulist.length}{" "}({$ulist.join(
+						", "
+					)}){/if}.
```

Comment:
Do we really need to include a list of users when we're going to remove this with the next release of server?
```suggestion
				There are currently {$ulist.length} users online.
				{/if}
```